### ISSUE TITLE
[12.0] Odoo client error on non-administrator user when no modules are installed

### ISSUE THREAD

**Issue by grzekru** [odoo/odoo#29503]
Impacted versions:

12.0

Steps to reproduce:

1. Create new database
2. Create new user without administrator permissions
3. Log into new user.
4. Error

Current behavior:

![image](https://user-images.githubusercontent.com/5862658/49931687-b5fcf100-fec7-11e8-8508-3f28d9af597a.png)

Expected behavior:

No error

Video/Screenshot link (optional):

On video i install custom theme but it throw error on normal odoo theme too.

https://www.youtube.com/watch?v=wi0MhU3-24M&feature=youtu.be

**Comment by pedrobaeza** [comment]
Hi, this seems something related to your deployment, as all the recent builds are correctly deployed in runbot (http://runbot.odoo.com/runbot), so please be sure you are following the proper installation instructions. Check them on https://www.odoo.com/documentation/12.0/setup/install.html.

I'm closing this issue now, but if you feel this can be something reproducible with other steps, please reply adding all the additional information for reproducing the problem.

**Comment by grzekru** [comment]
http://413991-12-0-7854c6.runbot11.odoo.com/web#id=52&action=68&model=res.users&view_type=form&menu_id=4

db: 413991-12-0-7854c6-base

login: test@test.pl
password: pass

**Comment by pedrobaeza** [comment]
And can you please tell me what are the steps to reproduce the problem?

**Comment by grzekru** [comment]
Can you reproduce this error on runbot?
Simply create new user before installing additional modules and than log in as new user.

**Comment by pedrobaeza** [comment]
Thanks to both for ditching into this. I'm reopening the issue, but I'm not sure who to ping, so I just tag it.

**Comment by Yenthe666** [comment]
@mart-e can you get this to the right person please?

**Comment by mart-e** [comment]
This is a web issue so I would say @ged-odoo or @aab-odoo team

PS: the error in debug=asset

```


TypeError: app is undefined

http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/chrome/apps_menu.js:70
Traceback:
_setActiveApp@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/chrome/apps_menu.js:70:13
_openApp@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/chrome/apps_menu.js:57:9
openFirstApp@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/chrome/apps_menu.js:45:9
openFirstApp@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/chrome/menu.js:199:9
show_application/</<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/chrome/web_client.js:103:29
then/</</<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3276:86
fire@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3119:38
fireWith@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3231:49
Deferred/</deferred[tuple[0]]@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3321:62
rpc/<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/services/ajax_service.js:16:17
then/</</<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3276:86
fire@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3119:38
fireWith@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3231:49
Deferred/</deferred[tuple[0]]@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3321:62
fire@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3119:38
fireWith@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3231:49
Deferred/</deferred[tuple[0]]@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3321:62
genericJsonRpc/<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/src/js/core/ajax.js:59:9
then/</</<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3276:86
fire@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3119:38
fireWith@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3231:49
then/</</<@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3283:104
fire@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3119:38
fireWith@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:3231:49
done@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:9275:33
callback@http://417050-12-0-ddc5cc.runbot15.odoo.com/web/static/lib/jquery/jquery.js:9685:57
```

**Comment by mart-e** [comment]
Or maybe something like #29687 :thinking:

**Comment by Elkasitu** [comment]
Fixed at #29687
